bundles/automation/org.eclipse.smarthome.automation.api/META-INF/MANIFEST.MF

```diff
@@ -5,6 +5,7 @@ Bundle-SymbolicName: org.eclipse.smarthome.automation.api
 Bundle-Version: 0.9.0.qualifier
 Bundle-Vendor: Eclipse.org/SmartHome
 Import-Package: 
+ com.google.common.base,
```

Comment:
@danchom I thought that you also wanted to avoid using Guava within the automation component, has this changed?